### ISSUE TITLE
Bug: taking notes does not work

### ISSUE THREAD

**Issue by 0-vortex** [open-sauced/open-sauced#955]
### To Reproduce

1. go to the dashboard
1. add any repository
1. click to view added repository
1. add a note
1. check the network tab
1. github request returned an error

### Expected behavior

Taking notes should work:
- added notes should exist on the "open-sauced-goals" repository as issues
- visually, after refresh the notes should still be there

### Screenshots

Sample JSON of the responses when querying:

```json
{
    "errors": [{
        "message": "Variable $id of type ID! was provided invalid value",
        "path": ["gitHub"],
        "extensions": {
            "service": "github",
            "upstream": {
                "errors": [{
                    "extensions": {
                        "value": null,
                        "problems": [{
                            "path": [],
                            "explanation": "Expected value to not be null"
                        }]
                    },
                    "locations": [{
                        "line": 2,
                        "column": 17
                    }],
                    "message": "Variable $id of type ID! was provided invalid value"
                }]
            },
            "traceId": "1eea08d3-b6d5-4c38-a1b0-cdb84b5d52ce"
        }
    }],
    "data": {
        "__typename": "Mutation",
        "gitHub": null
    }
}
```

### Desktop

- OS: all
- Browser: all
- Version: all

### Smartphone

- Device: all
- OS: all
- Browser: all
- Version: all

### Additional context

https://clips.twitch.tv/CuteDepressedPuffinWOOP-giOUmm9N_itsAVEo

**Comment by filiptronicek** [comment]
I could reproduce this, this worked just a couple months ago, so something must have broken it since then.

**Comment by mtfoley** [comment]
I've noticed it too. Similarly, going through "Delete Repository" (which I think just closes the issue in open-sauced-goals fork) doesn't work either.  From what I can tell, the problems come from not having the ID of the issue in the user's open-sauced-goals repository. For whatever reason, it's not making it into the React component's state for viewing the goal. I tried passing the ID in as a param in the Router, but that breaks other things for whatever reason. I can try to do a draft PR later for it...
